CHANGES.md

```diff
@@ -203,6 +203,8 @@ To be released.
     `RocksDBKeyValueStore.Dispose()` hadn't worked idempotently.  [[#1272]]
  -  Fixed a bug where `NetMQTransport` had hung forever within Mono runtime.
     [[#1278]]
+ -  Fixed a bug in `Ninechronicles.Headless` query API.
+    [[#1293], [NineChronicles.Headless #428]]
```

Comment:
```suggestion
    [[#1293]]
```
I'm not sure it is ok to mention project that this project does not have dependency of.

Reply:
As we haven't recorded changelogs for Libplanet Explorer, IMHO we'd better skip changelog for this pull request.